lunr/tokenizer.py

```diff
@@ -1,19 +1,24 @@
+from collections.abc import Sequence
 from copy import deepcopy
+from typing import Any, Callable, Dict, List, Union
 
 from lunr.token import Token
 from lunr.utils import as_string
 
 SEPARATOR_CHARS = " \t\n\r\f\v\xa0-"
 
 
-def default_separator(char):
-    return char and char in SEPARATOR_CHARS
+def default_separator(char: str) -> bool:
+    return char != "" and char in SEPARATOR_CHARS
```

Comment:
Added `!= ""` here because otherwise it will generally return a `str` not a `bool`.